cves/CVE-2014-0231.yml

```diff
@@ -4,7 +4,7 @@ CWE_instructions: |
   Please go to cwe.mitre.org and find the most specific, appropriate CWE entry
   that describes your vulnerability. (Tip: this may not be a good one to start
   with - spend time understanding this vulnerability before making your choice!)
-CWE:
+CWE: 754
```

Comment:
CWE-754: Improper Check for Unusual or Exceptional Conditions

I would think that a non terminating script is something to be expected for a scriptable system.

I think CWE-400: Uncontrolled Resource Consumption might be better.

Reply:
good catch! I think that is more appropriate, thank you.